### ISSUE TITLE
A11y_Code.visualstudio.com_Visual studio Search page_Search text field_Non text contrast: For "Search" field color contrast ratio between keyboard focus and background is < 3:1.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#5154]
## GitHub Tags:
#A11yTCS;#Win10;#WCAG1.4.11;#DesktopWeb;#A11ySev2;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;

## Environmental Details:
#URL: https://code.visualstudio.com/Search

## OS & Browser Details :
Microsoft Windows 10 Enterprise
Version 10.0.19044 Build 19044
Microsoft Edge
Version 99.0.1150.7 (Official build) dev (64-bit)

## Repro Steps :

1. Launch the URL https://code.visualstudio.com/Search
2. Tab till "Search" field
3. Verify whether the color contrast ratio between keyboard focus and background is >= 3:1 or not.

## Actual Result :
For "Search" field color contrast ratio between keyboard focus and background is < 3:1.

## Expected Result:
For "Search" field color contrast ratio between keyboard focus and background should be >=3:1.

## User Impact :
Low vision users may face difficulty to find the exact keyboard focus.

## Attachments:
![Bug_5154_Visual studio Search page_Search text field_Non text contrast](https://user-images.githubusercontent.com/99243079/153134424-9d007898-1cde-44e5-9bbb-22e09e323439.png)

**Comment by Samhitha66** [comment]
@gregvanl 
Checked again this bug in the below environment
URL: https://code.visualstudio.com/Search

Now , color contrast ratio between the keyboard focus and background for search field is > 3:1.
So, the issue is fixed.
![Bug_5154](https://user-images.githubusercontent.com/99243079/157194866-7b9a85aa-672c-4919-b7bf-41eb966fbd79.png)
